src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +115,13 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function setTemplateClass($templateClass)
+	{
+		\Nette\Utils\Validators::assert($templateClass, 'string');
+		if (!is_a($templateClass, UI\ITemplate::class, TRUE)) {
+			throw new \Nette\Utils\AssertionException('Template should be instance of ' . UI\ITemplate::class . ", $templateClass given.");
```

Comment:
* should be Nette\InvalidArgumentException, imho
* you need to call `!class_exist($templateClass) || ` before `!is_a` because `is_a` does not trigger autoloading